Assets/MixedRealityToolkit/MSBuild/Scripts/TargetFramework.cs

```diff
@@ -0,0 +1,76 @@
+﻿#if UNITY_EDITOR
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.MSBuild
+{
+    /// <summary>
+    /// Represents TargetFrameworks that Unity supports.
+    /// </summary>
+    public enum TargetFramework
+    {
+        NetStandard20,
+        Net20,
+        Net46
+    }
+    
+    /// <summary>
+    /// Helper extensions for the <see cref="TargetFramework"/> enum.
+    /// </summary>
+    public static class TargetFrameworkExtensions
+    {
+        /// <summary>
+        /// Converts a <see cref="TargetFramework"/> into an MSBuild acceptable string.
+        /// </summary>
+        /// <param name="this">The <see cref="TargetFramework"/> to convert.</param>
+        /// <returns>The MSBuild acceptable string representing the <see cref="TargetFramework"/>.</returns>
+        public static string AsMSBuildString(this TargetFramework @this)
+        {
+            switch (@this)
+            {
+                case TargetFramework.NetStandard20:
+                    return "netstandard2.0";
+                case TargetFramework.Net20:
+                    return "net20";
+                case TargetFramework.Net46:
+                    return "net46";
+            }
+
+            throw new ArgumentOutOfRangeException(nameof(@this));
+        }
+
+        /// <summary>
+        /// Returns the configured <see cref="TargetFramework"/> for the <see cref="BuildTargetGroup"/>.
+        /// </summary>
+        /// <param name="this">The <see cref="BuildTargetGroup"/> to get <see cref="TargetFramework"/> for.</param>
+        /// <returns>The <see cref="TargetFramework"/> configured for given <see cref="BuildTargetGroup"/>.</returns>
+        public static TargetFramework GetTargetFramework(this BuildTargetGroup @this)
+        {
+            if (@this == BuildTargetGroup.Unknown)
+            {
+                //TODO this may be different on older unity versions
```

Comment:
>//TODO this may be different on older unity versions [](start = 16, length = 52)

Is the plan to only support 2018 and 2019? Or is this just a general comment?

Reply:
Currently yes, and I don't know whether to leave this comment, or remove the "TODO" aspect of it


---
In reply to: [301803266](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5215#discussion_r301803266) [](ancestors = 301803266)